### PR TITLE
build: bump version to 0.9.0

### PR DESCRIPTION
**Pull request by JiaLiPassion**
We need to bump the version from `0.8` to `0.9` because we have some `minor API changes` in https://github.com/angular/zone.js/pull/1044, and we also add support for `evergreen` browsers.

### CODE REVIEW

**Comment by JiaLiPassion** [comment]
@IgorMinar, maybe we need to get a new release with a version bump to `0.9.0`, I also removed `Firefox 65` in this PR. please review, Thanks!

File: sauce-evergreen.conf.js [hunk 1]
```diff
@@ -7,7 +7,6 @@ module.exports = function(config, ignoredLaunchers) {
   var basicLaunchers = {
     'SL_CHROME': {base: 'SauceLabs', browserName: 'chrome', version: '72'},
     'SL_CHROME_60': {base: 'SauceLabs', browserName: 'chrome', version: '60'},
-    'SL_FIREFOX_59': {base: 'SauceLabs', browserName: 'firefox', platform: 'Windows 10', version: '65'},
```
Comment (IgorMinar) [inline sauce-evergreen.conf.js]: let’s move this into a separate commit

**Comment by JiaLiPassion** [comment]
@IgorMinar, got it! will revert the `Firefox` change. thanks!

**Comment by IgorMinar** [comment]
can you try to pull in terser and use it to create es2015 bundle for the evegreen browsers in a separate PR? Thank you

**Comment by JiaLiPassion** [comment]
@IgorMinar, sure, will do!

**Comment by JiaLiPassion** [comment]
I will create another PR.
